opm/parser/eclipse/EclipseState/Runspec.hpp

```diff
@@ -112,6 +112,69 @@ class WellSegmentDims {
     int nLatBranchMax;
 };
 
+class EclHysterConfig
+{
+public:
+      EclHysterConfig();
+      explicit EclHysterConfig(const Deck& deck);
+
+
+    /*!
+     * \brief Specify whether hysteresis is enabled or not.
+     */
+    void setEnableHysteresis(bool yesno);
+
+    /*!
+     * \brief Returns whether hysteresis is enabled.
+     */
+    const bool enableHysteresis() const;
```

Comment:
This class is already about hysteresis - no need for these long and verbose names; I would suggest:
```C++
void  enable(bool yesno);
bool  active() const;
```

And please don't add add comment strings for `setEnableHysteresis()` and `enableHysteresis()` - those comments are just noise.